bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -105,6 +105,9 @@ trait GemsStrategy extends AgsStrategy {
     cans ++ odgw
   }
 
+  // TODO Implement for GEMS
+  override protected [ags] def analyze(ctx: ObsContext, mt: MagnitudeTable, guideProbe: ValidatableGuideProbe, guideStar: SiderealTarget): Option[AgsAnalysis] = None
```

Comment:
Good question, it should be possible as it was done before, the `analyze` method in `AgsAnalysis` used to be called for GEMS and non-GEMS irrespective but I think it will default in GEMS to do an analyze of all the probes and return the same value for each guide star.
This should be part of the unfinished OCSADV-272 task